### PR TITLE
migrate print reports to b3

### PR DESCRIPTION
**Pull request by biyeun**
- clean up print reports to make sure they're not using old css and old js

@orangejenny / @benrudolph, maybe @esoergel 
cc @czue

### CODE REVIEW

**Comment by czue** [comment]
@kkrampa do you want to make sure these look ok?

File: corehq/apps/reports/templates/reports/async/print_report.html [hunk 1]
```diff
@@ -5,10 +5,10 @@
 {% if show_time_notice %}{% endif %}
 
 {% block js %}
+    <script src="{% static 'bootstrap/dist/js/bootstrap.min.js' %}"></script>
     <script src="{% static 'jquery/dist/jquery.min.js' %}"></script>
     <script src="{% static 'jquery-migrate/jquery-migrate.min.js' %}"></script>
-    {% include 'style/bootstrap2/partials/bootstrap2_js.html' %}
-    {{ block.super }}
+    <script type="text/javascript" src="{% static 'underscore-legacy/underscore-min.js' %}"></script>
```
Comment (benrudolph) [inline corehq/apps/reports/templates/reports/async/print_report.html]: doesn't this mean there are two underscores on the page being loaded?

**Comment by biyeun** [comment]
@czue @benrudolph  ✅

**Comment by czue** [comment]
lgtm - are there any concerns about testing the custom reports that this affects? feel free to merge if no (or if you tested)

**Comment by biyeun** [comment]
@czue there shouldn't be. none of those reports really utilized the JS to begin with...

**Pull request merged**